bootstrap.d/app-emulation.yml

```diff
@@ -0,0 +1,43 @@
+packages:
+  - name: qemu
+    architecture: '@OPTION:arch@'
+    metadata:
+      summary: A generic and open source machine emulator and virtualizer
+      description: A generic and open source machine emulator and virtualizer
+      spdx: 'GPL-2.0'
+      website: 'https://qemu.org/'
+      maintainer: 'Kacper Słomiński <qookie@managarm.org>'
+      categories: ['app-emulation']
+    source:
+      subdir: 'ports'
+      git: 'https://github.com/qemu/qemu.git'
+      tag: 'v6.2.0'
+      version: '6.2.0'
+    tools_required:
+      - host-pkg-config
+      - system-gcc
+      - virtual: pkgconfig-for-target
+        triple: "@OPTION:arch-triple@"
+    pkgs_required:
+      - mlibc
+      - glib
+      - sdl2
+      - pcre
+      - libiconv
+    configure:
+      - args:
+        - '@THIS_SOURCE_DIR@/configure'
+        - '--cross-prefix=@OPTION:arch-triple@-'
+        - '--host=@OPTION:arch-triple@'
+        - '--prefix=/usr'
+        - '--target-list=i386-softmmu,x86_64-softmmu,aarch64-softmmu'
```

Comment:
Maybe add the riscv target too for the fun, considering we're working on a riscv port too?

Reply:
Good point, enabled both riscv32 and riscv64 softmmu targets.